### DISCUSSION TITLE
Remove personal info keys from apis [Security]

### DISCUSSION THREAD

**Discussion by Stefanosapk** [cssbattle/forum#17] [Bugs reports]
Personal info keys such as email, etc..  is sensitive and never should be exposed.

An api that I identified that returns personal info is this: https://www.googleapis.com/identitytoolkit/v3/relyingparty/getAccountInfo?key

But needs to be removed and from anywhere else is exposed.

With any XSS vulnerability in the platform this can occur many issues to your users.

**Comment by chinchang** [comment]
Acknowledged! We'll work on this on priority. Thanks for reporting here 🙏🏼
